### PR TITLE
DB-3657: Block Editor Video Styles

### PR DESCRIPTION
**Pull request by mitchellmarkoff**
## What changes were made?

- Added Video component

## Where were the changes made?
`wordpress-kit`

## How have the changes been tested?
Implemented this component into the next-wordpress-starter locally and tested a sample post with a video to observe expected styles and behavior.

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

**Review by CobyPear — CHANGES_REQUESTED** [review]
Seems to be working for an uploaded video, but not an embedded one.
@backlineint would you say that is in scope of this ticket or handled in the future?

**Comment by mitchellmarkoff** [comment]
> Seems to be working for an uploaded video, but not an embedded one. @backlineint would you say that is in scope of this ticket or handled in the future?

Yeah the way WP seems to handle embedded vs uploaded videos is pretty different. Uploaded videos are put into a `vdieo` tag and given the `wp-block-video `class. Embedded videos are placed into a `div` and not passed the `wp-block-video-class`.

**Comment by backlineint** [comment]
> > Seems to be working for an uploaded video, but not an embedded one. @backlineint would you say that is in scope of this ticket or handled in the future?
> 
> Yeah WP seems to handle embedded and uploaded videos is pretty differently. Uploaded videos and embedded videos do not share block editor classes.

Wasn't aware that the block editor handled them differently. Unfortunately I think embedded is the more important use case here, so I'd say we should just continue work on this one @mitchellmarkoff

**Review by CobyPear — CHANGES_REQUESTED** [review]
The block style looks good! Looks like a change for testing got pushed up that needs to be removed.

File: starters/next-wordpress-starter/tailwind.config.js [hunk 1]
```diff
@@ -7,7 +7,8 @@ module.exports = {
 		'./components/**/*.{js,ts,jsx,tsx}',
 	],
 	theme: {
-		extend: {},
+		// colors: { extend: 'red' }
+		extend: { colors: { extend: 'yellow' } },
```
[the remark this answers is not in the document]
Comment (mitchellmarkoff) [reply starters/next-wordpress-starter/tailwind.config.js:11]: Yes good catch!

**Pull request merged**